docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,29 @@ kotlin {
         }
     }
 }
+```
+
+### Dependencies between Pods
```

Comment:
I think it's better not to use the plain "dependency" term. Because:
1) That's not really correct technically because dependency is added not between pods themselves but between their C-interops
2) That sounds quite simple, but the case for which it is needed is complicated. I think users can misinterpret that
Due to these reasons, the functions in the DSL are not called "dependsOn" and "dependencies"

I'd suggest smth like "Share Kotlin C-interop between dependant pods". But I don't know if we're ok of using C-interop term because it's quite technical.

Reply:
I think it's OK. We're using "cinterop" a lot in the context of ios dependencies